dev/src/serializer.ts

```diff
@@ -43,7 +44,7 @@ export class Serializer {
   private createReference: (path: string) => AnyDuringMigration;
 
   constructor(
-      firestore: AnyDuringMigration,
+      private readonly firestore: Firestore,
```

Comment:
This looks like it's adding a field. Is that necessary? Do you actually reference it anywhere?

Reply:
Ahhhh. That's probably why the JSON.stringify() failed. Thanks for catching this.